packages/amplication-git-utils/src/providers/bitbucket/bitbucket.service.ts

```diff
@@ -526,7 +527,7 @@ export class BitBucketService implements GitProvider {
     );
   }
 
-  async getToken(): Promise<string> {
+  private async getToken(): Promise<string> {
```

Comment:
Is this needed at all? the accessToken should be a property of the service already without refreshing it every time